src/frontend/src/components/DesignReviewPill.tsx

```diff
@@ -0,0 +1,35 @@
+import { Typography, Link } from '@mui/material';
+
+// component for the DR pills that are links (zoom and docs)
+export const DesignReviewPill: React.FC<{
```

Comment:
these two components should go in a folder next to the other files you added rather than in the components folder